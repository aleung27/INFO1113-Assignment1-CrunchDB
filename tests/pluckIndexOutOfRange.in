SET y 2 1 4 5 5 5 5 8
PLUCK y 0
PLUCK y 9
PLUCK y 8
PLUCK y 8
PLUCK y 0
PLUCK y 7
LIST ENTRIES
