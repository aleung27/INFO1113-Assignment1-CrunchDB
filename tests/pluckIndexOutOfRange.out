> ok

> index out of range

> index out of range

> 8

> index out of range

> index out of range

> 5

> y [2 1 4 5 5 5]

